pom.xml

```diff
@@ -401,6 +401,12 @@
 				<version>6.0.0</version>
 				<scope>provided</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.osgi</groupId>
+				<artifactId>org.osgi.service.cm</artifactId>
+				<version>1.6.0</version>
+				<scope>provided</scope>
```

Comment:
This is clearly NOT provided. The OSGI framework does not export the cm package at runtime.